### PR TITLE
Add port config to deployment spec.

### PR DESCRIPTION
**Pull request by telackey**
After the change, ports as well as volumes will be customizable in the "spec" output from `deploy init`.

Example:

```
stack: mainnet-eth
ports:
  mainnet-eth-geth-1:
   - '8545'
   - '40000'
   - '6060'
  mainnet-eth-lighthouse-1:
   - '8001'
volumes:
  mainnet_eth_config_data: ./data/mainnet_eth_config_data
  mainnet_eth_geth_1_data: ./data/mainnet_eth_geth_1_data
  mainnet_eth_lighthouse_1_data: ./data/mainnet_eth_lighthouse_1_data
```

### CODE REVIEW

**Comment by dboreham** [comment]
> After the change, ports as well as volumes will be customizable in the "spec" output from `deploy init`.
> 
> Example:

Can you update the example?

**Comment by dboreham** [comment]
Can you add host address/port mapping to one or more of the ports in the example so we can use it for doc?

File: app/deployment_create.py [hunk 1]
```diff
@@ -148,12 +168,18 @@ def init(ctx, output):
         spec_file_content.update(default_spec_file_content)
     if verbose:
         print(f"Creating spec file for stack: {stack}")
+
+    ports = _get_named_volumes(stack)
```
Comment (dboreham) [inline app/deployment_create.py:172]: Something looks wrong about this line.

Comment (telackey) [reply app/deployment_create.py:172]: Hah!  That's what I get for making a last minute tweak.

**Review by dboreham — APPROVED** [review]

**Pull request merged**